src/core/js/router.js

```diff
@@ -206,6 +207,20 @@ define([
             this.handleRoute.apply(this, args);
         },
 
+        skipNavigation: function() {
+
+            var pageCount = $(".page").length;
+            var pageContent = $(".page-inner");
+            var menuContent = $(".menu");
+
+            if (pageCount == 1) {
+                pageContent.a11y_focus();
+            } else {
+                menuContent.a11y_focus();
+            }
+
```

Comment:
core template also needs updating as per pr to vanilla https://github.com/adaptlearning/adapt-contrib-vanilla/pull/201